HIP 138: Return to HNT (#1119)
helium/HIP: 0138-return-to-hnt.md

# HIP 138: Return to HNT

- Author: [@ferebee](https://github.com/ferebee), [@JMF](https://github.com/JMFayal), [@abhay](https://github.com/abhay)

> - Start Date: 2024-11-06

Reviewer: ```suggestion
- Start Date: 2024-11-08
```

- Category: Economic, Technical, Governance

> - Original HIP PR: [#XXXX](https://github.com/helium/HIP/pull/XXXX)

Reviewer: ```suggestion
- Original HIP PR: [#1119](https://github.com/helium/HIP/pull/1119)
```

> - Tracking Issue: [#XXXX](https://github.com/helium/HIP/issues/XXXX)

Reviewer 2: ```suggestion
- Tracking Issue: [#1120](https://github.com/helium/HIP/issues/1120)
```

- Voting Requirements: veHNT Holders

---

## Summary

> If approved and implemented, this proposal would phase out the subnetwork tokens introduced through HIPs [51][hip-51], [52][hip-52], and [53][hip-53] (IOT and MOBILE) and the network would return to rewarding participants directly in HNT. Additionally, HNT emissions allocated to HST holders would end and be redirected to the MOBILE treasury until the next halvening (August 1, 2025). Post halvening, these rewards will be issued, like existing emissions, to Hotspot owners and other network participants via the Utility Score. Finally, unissued HNT (approximately 4.2 million) from the Helium L1 would additionally fund the MOBILE treasury.

Reviewer: ```suggestion
If approved and implemented, this proposal would phase out the subnetwork tokens introduced through HIPs [51][hip-51], [52][hip-52], and [53][hip-53] (IOT and MOBILE) and the network would return to rewarding participants directly in HNT. Additionally, HNT emissions allocated to HST holders would end and be redirected to the MOBILE treasury until the next halvening (August 1, 2025). Post halvening, these rewards will be issued, like existing emissions, to Hotspot owners and other network participants via the Utility Score. Finally, unissued HNT (approximately 4.2 million) from the Helium L1 would gradually fund the MOBILE treasury.
```

> Moving to a single token simplifies the Helium ecosystem, making it more understandable for current and future network builders and more accessible to the crypto community at large. Using HNT as the sole reward currency also corrects the imbalances created by the treasury system. Specifically, MOBILE network participants would see immediate increases in the value of their rewards, while IOT participants’ rewards would continue to be supported post-halvening as the network grows through ongoing community investment.

Reviewer: ```suggestion
Moving to a single token simplifies the Helium ecosystem, making it more understandable for current and future network builders and more accessible to the crypto community at large. Using HNT as the sole reward currency also corrects the imbalances created by the treasury system. Specifically, MOBILE network participants would see an immediate correction of the underlying redemption rate of HNT, while IOT participants’ rewards would continue to be supported post-halvening as the network grows through ongoing community investment.
```

### Builder Rewards

As the MOBILE network matures, it’s evident that MOBILE rewards are valued lower in HNT terms than the HNT emissions allocated to the MOBILE treasury. If MOBILE network builders received direct HNT rewards instead, their rewards would increase by about 2.5 times in HNT terms, creating a strong incentive for continued network development.

Meanwhile, IOT’s value is closely aligned with its treasury support, and it remains a vital part of the ecosystem. This proposal seeks to maintain that support.

## Stakeholders

All Helium Ecosystem participants are affected by this proposal.

## Detailed Explanation

This proposal suggests several tokenomic changes to simplify the Helium ecosystem and promote future network development.

At a high level, we propose to:

1. Discontinue HST emissions beginning with the start of “Phase 1” (defined below), redirecting HST emissions to the MOBILE treasury. This would continue until the next halvening.
2. Discontinue IOT and MOBILE Emissions at the start of “Phase 2” (also defined below) and reward participants directly in HNT based on the Utility Score.

> 3. Recognize the 4.2 million unissued HNT from Helium L1 after HIP-20 and begin emitting these tokens directly into the MOBILE treasury over time throughout “Phase 2” until the next halvening.

Reviewer: ```suggestion
3. Recognize the 4.2 million unissued HNT from Helium L1 after [HIP-20][hip-20] and begin emitting these tokens directly into the MOBILE treasury over time throughout “Phase 2” until the next halvening.
```

Rewards will be distributed according to the existing subnetwork structure and remain subject to subnetwork governance. For example, the 6% of subnetwork token emissions rewarded to delegators will instead be rewarded as 6% of the HNT awarded to that subnetwork.

Emission shares which are unallocated (such as Oracle rewards) or partially unallocated (potentially MOBILE Service Provider rewards) will be burned unless a future HIP specifies otherwise.

### Treasuries

Treasuries retain their current HNT balances. Existing IOT and MOBILE wallet balances remain unchanged, and holders can continue to hold, or can redeem for HNT at any time at the treasury rate.

### Value Imbalances

Due to market factors, the IOT treasury is well-funded relative to IOT network contributions, whereas a large portion of early MOBILE rewards were minted without a backing of treasury HNT.

Currently, IOT trades at its treasury rate to HNT, while MOBILE trades at a speculative premium, possibly reflecting expected future value. As a result, current rewards and valuations don’t fully match the underlying HNT emissions.

> 1.	Higher Value of IOT Emissions: IOT rewards are valued higher in HNT than the HNT emitted to the IOT network, based on Utility Score.
> 2.	Current Strong Exchange Rate for IOT Tokens: IOT converts to HNT at a strong rate, though this may decline as MOBILE network revenue increases, and HNT emissions to IOT decrease.
> 3.	Lower Value of MOBILE Emissions: MOBILE rewards are worth less in HNT than the HNT emitted to the MOBILE subnetwork, due to lower HNT funding of the MOBILE treasury relative to subnetwork utility.
> 4.	Market Premium on MOBILE Tokens: MOBILE trades 1.4 times higher than its treasury rate.

Reviewer: ```suggestion
1. Higher Value of IOT Emissions: IOT rewards are valued higher in HNT than the HNT emitted to the IOT network, based on Utility Score.
2. Current Strong Exchange Rate for IOT Tokens: IOT converts to HNT at a strong rate, though this may decline as MOBILE network revenue increases, and HNT emissions to IOT decrease.
3. Lower Value of MOBILE Emissions: MOBILE rewards are worth less in HNT than the HNT emitted to the MOBILE subnetwork, due to lower HNT funding of the MOBILE treasury relative to subnetwork utility.
4. Market Premium on MOBILE Tokens: MOBILE trades 1.4 times higher than its treasury rate.
```

If subnetwork emissions were to shift to HNT without adjustments, potential effects would be:

- IOT network builders may see reduced HNT value of rewards.
- IOT holders would benefit, with a fixed IOT:HNT treasury rate.
- MOBILE network builders would benefit, with HNT rewards valued 2.5 times current MOBILE rewards.
- MOBILE holders would lose speculative premium, reducing their holdings’ value closer to the treasury rate.

This proposal introduces provisions to counteract these potential disadvantage.

### Subnetwork Governance

> Subnetwork governance remains in force, and this proposal doesn’t modify the existing system of governance using veIOT and veMOBILE, as derived from locked IOT and MOBILE tokens. While no new subnetwork tokens will be minted, participants can acquire voting rights by purchasing tokens on the open market and locking them.

Reviewer: ```suggestion
Subnetwork governance remains in force, and this proposal doesn't modify the existing system of governance using veIOT and veMOBILE, as derived from locked IOT and MOBILE tokens. While no new subnetwork tokens will be minted, participants can acquire voting rights by purchasing tokens on the open market and locking them.
```

#### MOBILE Treasury Adjustment

Given that the MOBILE treasury is underfunded by a factor of 1.4 relative to the market value of MOBILE, this proposal redirects HST emissions to the MOBILE treasury from the beginning of Phase 1 until the halvening, thereby improving the treasury rate over time. If Phase 1 were to begin on January 1, this would be a subsidy of approximately 2.6M HNT.

#### IOT Rewards Post Halvening

During Phase 2, the HNT rewarded to IOT participants may be less than the HNT value of the rewards previously emitted as IOT tokens, as long as the surplus funding of the IOT treasury has not yet been fully redeemed through treasury swaps.

To compensate for this, the additional HNT emissions from HST which are redirected towards the IOT and MOBILE ecosystem beginning with the halvening, comprising 29% of total HNT emissions in the year beginning on August 1, 2025, will be shared between the IOT and MOBILE subnetworks according to the DAO Utility Score, and the Utility Score as specified in HIP-51, including its A factor, will remain in effect.

### Unissued HNT Adjustment

> Following HIP-20, Helium experienced L1 outages, leaving 4.2 million unminted HNT. This proposal mints these 4.2 million HNT to the MOBILE treasury from the start of Phase 2 until the halvening.

Reviewer: ```suggestion
Following HIP-20, Helium experienced L1 outages, leaving 4.2 million unminted HNT. This proposal gradually mints these 4.2 million HNT to the MOBILE treasury from the start of Phase 2 until the halvening.
```
